docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -61,6 +61,21 @@ Elliptic Curve Signature Algorithms
     :func:`~cryptography.hazmat.primitives.asymmetric.utils.decode_dss_signature`.
 
 
+    Verification requires the public key, the signature itself, the signed data and knowledge of the hashing algorithm that was used when producing the signature:
```

Comment:
Nit: could you put a comma after `the signed data`. We generally use Oxford comma in our docs.